### PR TITLE
Register new package Phylogenies.jl v0.1.0

### PR DESCRIPTION
**Pull request by attobot**
Repository: [BioJulia/Phylogenies.jl](https://github.com/BioJulia/Phylogenies.jl)
Release: [v0.1.0](https://github.com/BioJulia/Phylogenies.jl/releases/tag/v0.1.0)
Travis: [Travis Build Status](https://travis-ci.org/BioJulia/Phylogenies.jl/builds/216767097)
cc: @Ward9250

### CODE REVIEW

**Comment by Ward9250** [comment]
This is a next step in the compartmentalisation of the code in Bio.jl, removing Bio.Phylo into Phylogenies.jl

File: Phylogenies/versions/0.1.0/requires [hunk 1]
```diff
@@ -0,0 +1,4 @@
+julia 0.5
+Compat 0.17.0
```
Comment (tkelman) [inline Phylogenies/versions/0.1.0/requires]: I don't see any calls to the `@compat` macro, are you relying on one of the method extensions?

Comment (Ward9250) [inline Phylogenies/versions/0.1.0/requires]: Thought I'd forgotten something, I was using compat for using "where" syntax for some parametric methods, but couldn't quite figure it out, and thought it wasnt worth it - I'm changing code in Phylogenies as my next project anyway and when I do 0.6 may well be here anyway.

**Comment by Ward9250** [comment]
Ok removed compat.

**Comment by tkelman** [comment]
I still see a few `using Compat` in src

**Comment by tkelman** [comment]
Distributions is also missing from `REQUIRE`, used here https://github.com/BioJulia/Phylogenies.jl/blob/cb9ea094d07c27559b681a779d9d87950e3e6f8a/src/dating.jl#L12

**Comment by Ward9250** [comment]
Dammit! disentangling modules is a nuisance, ok that should be compat gone.

**Comment by Ward9250** [comment]
:+1: Thanks!

**Pull request merged**